flake.nix

```diff
@@ -1,16 +1,43 @@
 {
   inputs = {
-    core.url = "github:auxolotl/packages";
+    core.url = "github:auxolotl/core";
 
     javascript = {
       url = "github:auxolotl/javascript";
-      inputs.top-level.follows = "";
+      inputs.core.follows = "core";
     };
   };
 
-  outputs = { self, core, javascript }: {
-    inherit core;
+  outputs =
+    {
+      self,
+      core,
+      javascript,
+    }:
+    let
+      forAllSystems =
```

Comment:
Given we have the same function in the [JS repo](https://github.com/auxolotl/javascript/blob/main/flake.nix#L9-L17), I think it might be worth moving this function into core if possible. I don't think that should block merging for now though.